packages/ProgressIndicators/src/ProgressBar/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+/** Props for ProgressBar */
+interface ProgressBarProps {
+  animate: boolean /** apply animation to progressBar */;
+  decimalPoints: number /** Rounds off the value to this number */;
+  height: string /** resizes the height of the progressBar */;
+  min: number /** set lower bound for the progressBar range */;
+  max: number /** set upper bound for the progressBar range */;
+  progressStrippedClass: string /** set progressBar class to stripped */;
```

Comment:
How do you feel about providing one prop for the css class to be applied?  The person using the component will then give whatever full string he wants

Reply:
I think I agree with that